examples/channel-synchronization/channel-synchronization.go

```diff
@@ -1,6 +1,8 @@
 // We can use channels to synchronize execution
 // across goroutines. Here's an example of using a
 // blocking receive to wait for a goroutine to finish.
+// When waiting for multiple goroutines to finish,
+// you may prefer to [use a WaitGroup](waiting-for-goroutines-to-finish).
```

Comment:
```suggestion
// you may prefer to use a [WaitGroup](waiting-for-goroutines-to-finish).
```

(as you do in the worker pools example.)